libr/include/r_core.h

```diff
@@ -243,6 +243,13 @@ typedef struct r_core_tasks_t {
 	bool oneshot_running;
 } RCoreTaskScheduler;
 
+typedef struct r_core_graph_hits_t {
```

Comment:
why here and not in `RAGraph`? It seems something more related to RAGraph than to Core in general.

Reply:
fxed